### ISSUE TITLE
Cannot install on Ubuntu 22.10

### ISSUE THREAD

**Issue by hughjdavey** [mozilla-mobile/mozilla-vpn-client#4705]
I think this is because `systemd-resolved` has replaced `resolvconf` in some way.

Try to install `mozillavpn` as normal:

```shell
hugh@hugh:~$ sudo apt-get install mozillavpn 
[sudo] password for hugh: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 mozillavpn : Depends: resolvconf (>= 1.84ubuntu1)
E: Unable to correct problems, you have held broken packages.
```

Then try to install `resolvconf` as well:

```shell
hugh@hugh:~$ sudo apt-get install resolvconf mozillavpn
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'systemd-resolved' instead of 'resolvconf'
systemd-resolved is already the newest version (251.4-1ubuntu7).
systemd-resolved set to manually installed.
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 mozillavpn : Depends: resolvconf (>= 1.84ubuntu1)
E: Unable to correct problems, you have held broken packages.
```

See `resolvconf` version:

```shell
hugh@hugh:~$ resolvconf --version
systemd 251 (251.4-1ubuntu7)
+PAM +AUDIT +SELINUX +APPARMOR +IMA +SMACK +SECCOMP +GCRYPT -GNUTLS +OPENSSL +ACL +BLKID +CURL +ELFUTILS +FIDO2 +IDN2 -IDN +IPTC +KMOD +LIBCRYPTSETUP +LIBFDISK +PCRE2 -PWQUALITY -P11KIT -QRENCODE +TPM2 +BZIP2 +LZ4 +XZ +ZLIB +ZSTD -BPF_FRAMEWORK -XKBCOMMON +UTMP +SYSVINIT default-hierarchy=unified
```

Is there any workaround or fix to this that you know of? Thanks for your time

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-3064)
┆Link To Issue: https://mozilla-hub.atlassian.net/browse/VPN-3064

**Comment by bakulf** [comment]
Ubuntu 22.10 is not officially supported yet. As soon as we release MozillaVPN 2.10, we will release ubuntu kinetic packages too

**Comment by hughjdavey** [comment]
Thanks @bakulf - do you know when that will be?

**Comment by bakulf** [comment]
probably this week.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Closing this bug as this was addressed in: https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4713 ( https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4713|smart-link )

Please feel free to reopen if it’s still not working!

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Adding the December sprint to this ticket.

**Comment by hughjdavey** [comment]
Hi

This still doesn't work. [The docs](https://support.mozilla.org/en-US/kb/how-install-mozilla-vpn-linux-computer) only mention support for up to 22.04 and I am unable to install on 22.10:

```shell
hugh@hugh:~$ sudo apt install mozillavpn 
[sudo] password for hugh: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 mozillavpn : Depends: resolvconf (>= 1.84ubuntu1)
E: Unable to correct problems, you have held broken packages.
hugh@hugh:~$ # follow resolvconf tip from docs
hugh@hugh:~$ sudo apt install resolvconf mozillavpn
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'systemd-resolved' instead of 'resolvconf'
systemd-resolved is already the newest version (251.4-1ubuntu7).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 mozillavpn : Depends: resolvconf (>= 1.84ubuntu1)
E: Unable to correct problems, you have held broken packages.
```

Thanks for your help
